python/ray/util/accelerators/types.py

```diff
@@ -0,0 +1,103 @@
+import ray.util.accelerators.accelerators as _accelerators
+from ray.util.annotations import DeveloperAPI, PublicAPI
+
+
+@DeveloperAPI(stability="alpha")
+class AcceleratorTypes:
+    """Unified access point for all accelerator types supported by Ray, categorized by vendor.
```

Comment:
This will never be a complete list of all accelerator types supported by Ray since new accelerators will be introduced. Instead, we should document how the accelerator type is decided and how the user can see the accelerator type string of each node and that's the string they can use for `accelerator_type` option.